keyboards/8pack/readme.md

```diff
@@ -0,0 +1,13 @@
+# 8-Pack Macropad
+
+An open source 2x4 macropad designed by Charles Garcia.
+
+Keyboard Maintainer: [Charles Garcia](https://github.com/cgarcia2097)
```

Comment:
We need to add double space to end of lines in order for github to render the file properly

```suggestion
Keyboard Maintainer: [Charles Garcia](https://github.com/cgarcia2097)  
```

Reply:
I'm not a huge fan of double space, as editors have a habit of lopping it off. Much prefer explict <br>

Reply:
look at that, it does indeed not render properly. I'll add the double spaces once I get back home after work.